src/styles/blocks/on-view.scss

```diff
@@ -0,0 +1,163 @@
+.on-view {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__title {
+    font-size: 32px;
+    font-weight: 700;
+    line-height: 32px;
+    letter-spacing: -1.5px;
+    text-align: center;
+    color: #161616;
+
+    @media (min-width: 1024px) {
+      font-size: 48px;
+    }
+  }
+
+  &__subtitle {
+    padding-top: 8px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 27px;
+    text-align: center;
+  }
+
+  &__gallery {
+    display: flex;
+    flex-direction: column;
+    padding-top: 48px;
+    justify-content: center;
+
+    &-image {
+      img {
+        width: 130px;
+        height: 130px;
+      }
+
+      @media (min-width: 1024px) {
+        img {
+          width: 324px;
+          height: 320px;
+        }
+      }
+    }
+
+    &-image--wide {
+      img {
+        width: 130px;
+        height: 130px;
+        object-fit: cover;
+      }
+
+      @media (min-width: 1024px) {
+        img {
+          width: 672px;
+          height: 320px;
+        }
+      }
+    }
+
+    &-1 {
+      display: flex;
+      gap: 24px;
+
+      .on-view__gallery-image--wide {
+        width: 130px;
+        height: 130px;
+        background-size: cover;
+        background-image: url('/src/images/portrait.png');
+
+        & img {
+          display: none;
+        }
+
+        @media (min-width: 1024px) {
+          width: 672px;
+          height: 320px;
+          & img {
+            display: block;
+          }
+        }
+      }
+    }
+
+    &-2 {
+      display: flex;
+      gap: 8px;
+
+      & .on-view__gallery-image img {
+        width: 88px;
+        height: 87px;
+      }
+
+      @media (min-width: 1024px) {
+        gap: 20px;
+        img {
+          width: 324px;
+          height: 320px;
+        }
+      }
```

Comment:
Your style here is not matching the Figma. It does not need to be pixel perfect, but it's recommended to be as close as possible to the Figma design to ensure a neat and cohesive appearance.